### PR TITLE
Create models for friend and friend request

### PR DESCRIPTION
**Pull request by sp35**
created models

### CODE REVIEW

File: app/models.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
 from django.contrib.auth.models import AbstractBaseUser
 from django.db import models
+from django.contrib.auth.models import User
```
Comment (KIRA009) [inline app/models.py]: You don't need to use this User model. We already have a User model defined in models.py file

File: app/models.py [hunk 2]
```diff
@@ -15,3 +17,41 @@ class User(AbstractBaseUser):
...
+        on_delete=models.CASCADE,
+        related_name="user_sent",
+    )
+    to_user = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        related_name="user_received",
+    )
+
+    message = models.TextField(blank=True)
```
Comment (KIRA009) [inline app/models.py]: Could you explain the use of this field?

**Review by KIRA009 — COMMENTED** [review]
Try naming your commits and pull requests in a more informative way
For example, in this case, you could have had something like this

[feat] Add models for friends and friend requests

You can refer <a href="https://chris.beams.io/posts/git-commit/">this</a> link for better naming


Also, I have updated the issues page. Please check which issues are getting solved by your pr and mention them like this

This solves #1, #2 and #3

where 1, 2, and 3 are the issue numbers, which you can find in the respective issues page

File: app/models.py (hunk 2 above)
Comment (sp35) [reply app/models.py]: I thought we could use to send custom friend request messages else a default request message.

File: app/models.py (hunk 1 above)
Comment (sp35) [reply app/models.py]: Oh! my bad. I'll remove this.

**Comment by sp35** [comment]
This solves #4

File: app/models.py (hunk 2 above)
Comment (KIRA009) [reply app/models.py]: Remove it for now, we can add it later if needed

**Comment by KIRA009** [comment]
Okay, everything seems fine as of now. Just check all the unresolved conversations, make the changes, then pull rebase and push. If you have any doubts at any step, before doing that, please contact me

**Pull request merged**